tests/unit/plugins/modules/net_tools/test_dnsimple_info.py

```diff
@@ -0,0 +1,25 @@
+__metaclass__ = type
+import pytest
+import traceback
+import sys
+
+try:
+    import dnsimple_info
+    from requests import Request, Session
+    from community.general.plugins.modules.net_tools import dnsimple_info
```

Comment:
```suggestion
    from ansible_collections.community.general.plugins.modules.net_tools import dnsimple_info
```
Also this needs to be outside the `try`/`except` block. The whole block isn't needed anyway, see my link to the dnsimple tests.